### PR TITLE
Revise section "Zope Security"

### PR DESCRIPTION
**Pull request by jugmac00**
- improve formatting
- add links for further information about Acquisition
- fix references to other chapters

modified:   docs/zdgbook/ObjectPublishing.rst

### CODE REVIEW

**Comment by jugmac00** [comment]
I have to admit, this is one of the first section which I revise, which I am not able to follow / understand completely. Especially the first half sentence. Like, of course we use Zope - its the application server. I do not get the difference between "using Zope" vs "own module".

Also, I am unsure whether the "Explicit or Implicit" is necessary is correct.

For example, in the legacy code base I got a class, which only inherits from "OFS.Folder.Folder".

Having a look at the mro:
```(<class 'OFS.Folder.Folder'>,
 <class 'OFS.ObjectManager.ObjectManager'>,
 <class 'OFS.CopySupport.CopyContainer'>,
 <class 'App.Management.Navigation'>,
 <class 'App.Management.Tabs'>,
 <type 'Acquirer'>,
 <type 'Persistence.Persistent'>,
 <type 'persistent.Persistent'>,
 <class 'webdav.Collection.Collection'>,
 <class 'webdav.Resource.Resource'>,
 <class webdav.Lockable.LockableItem at 0x7f61135d7a78>,
 <class webdav.EtagSupport.EtagSupport at 0x7f61135d79a8>,
 <class OFS.Traversable.Traversable at 0x7f6112bd1ae0>,
 <class 'OFS.PropertyManager.PropertyManager'>,
 <class OFS.ZDOM.ElementWithAttributes at 0x7f6112bf6b48>,
 <class OFS.ZDOM.Element at 0x7f6112bf6ae0>,
 <class OFS.ZDOM.Node at 0x7f6112bf6a10>,
 <class 'OFS.role.RoleManager'>,
 <class 'AccessControl.rolemanager.RoleManager'>,
 <class AccessControl.PermissionMapping.RoleManager at 0x7f6112f07808>,
 <class 'OFS.SimpleItem.Item'>,
 <class 'OFS.CopySupport.CopySource'>,
 <class 'OFS.owner.Owned'>,
 <class 'AccessControl.owner.Owned'>,
 <class 'App.Undo.UndoSupport'>,
 <class 'OFS.FindSupport.FindSupport'>,
 <type 'ExtensionClass.Base'>,
 <type 'object'>)```

It neither inherits from Explicit nor Implicit, but still gets published.

Any suggestions on this?

**Comment by dataflake** [comment]
>  Like, of course we use Zope - its the application server. I do not get the difference between "using Zope" vs "own module".

Originally the ZPublisher was thought of a generic publisher that could publish not just Zope, but other modules as well. IMHO that whole half sentence can go away. Instead of `When using Zope rather than publishing your own modules, the` replace it eith `The`.

> Having a look at the mro:

I have never used the mro to determine subclasses/mixins, so I can't say whether that is a suitable approach at all. Instead, look at the source, Luke! ``OFS.Folder.Folder`` derives from ``OFS.ObjectManager.ObjectManager``, which mixes in ``Acquisition.Implicit``.

File: docs/zdgbook/ObjectPublishing.rst [hunk 1]
```diff
@@ -566,25 +566,42 @@ Zope Security
 When using Zope rather than publishing your own modules, the publisher
```
Comment (dataflake) [inline docs/zdgbook/ObjectPublishing.rst]: Replace `When using Zope rather than publishing your own modules, the` with just `The`

**Comment by jugmac00** [comment]
Once again, thanks a lot for taking your time to review my changes and the hint about `OFS.ObjectManager.ObjectManager`!

“I’m just a simple man trying to make my way in the universe.” — Jango Fett

**Comment by dataflake** [comment]
@jugmac00 I'm really happy you are taking this time to rework the docs. You know best how short-staffed we are. @icemac will agree that any help is **highly** appreciated.

**Comment by jugmac00** [comment]
@dataflake Would you mind to approving the pull request? :-)

**Comment by dataflake** [comment]
Oops, had not seen the last commit

**Comment by icemac** [comment]
Help is highly appreciated especially on the documentation which was not updated in years.

**Pull request merged**